config/ADO.yml

```diff
@@ -0,0 +1,22 @@
+idspace: ADO
+base_url: /obo/ado
+
+products:
+- ado.owl: https://raw.githubusercontent.com/Fraunhofer-SCAI-Applied-Semantics/ADO/main/Releases/2.0.0/ADO.owl
```

Comment:
This is a redirect to a version iri. There should be one versioned release, and one non versioned release (you don't want to have to edit this file every time you create a new version.

I would suggest this:

1. You remove `Releases/2.0.0/ADO.owl` and simply add a `ado.owl` (lower case!)
2. then whenever you make a release, you run a github release like you do [here](https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/releases)
3. You can then simply refer to ado.owl through the GitHub release tag in this purl config (I will show you how)

Reply:
Thank you for your reply. I have changed the file name as well as the products link. The link does not resolve now.

Reply:
Ok, one more thing required, and then you are done:

- [ ] move Releases/2.0.0/ADO.owl in your repo to ado.owl (top level) - delete the `Releases` folder altogether because GitHub releases will take care of your versioning.

Reply:
- [ ] ado.owl must be lower case.
- [ ] you cant just delete a release, you also have to delete the associated tag. then redo the release.